styles/google/catppuccin.user.css

```diff
@@ -474,6 +474,14 @@
 
     /* End of AI search */
 
+    /* Featured snippets */
+    @media (prefers-reduced-motion: no-preference) {
+      .c2xzTb .ILfuVd b {
+        background: linear-gradient(90deg, fade(@accent-color, 30%) 50%, #0000 50%) no-repeat 75% 0;
+        background-size: 200% 100%;
+      }
+    }
```

Comment:
Based these off of google's own css for this element.
<img width="1219" alt="Screenshot 2024-08-05 at 7 50 37 a m" src="https://github.com/user-attachments/assets/8769b240-c160-4e1d-a842-c151e132d662">